### PR TITLE
Move supportFeature to mcdata

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by u9g** [comment]
requires https://github.com/PrismarineJS/node-minecraft-data/pull/162

**Comment by rom1504** [comment]
can you also remove features.json from here ?

**Comment by u9g** [comment]
> can you also remove features.json from here ?

Done

File: lib/loader.js [hunk 1]
```diff
@@ -56,7 +55,7 @@ module.exports = {
   BossBar: require('./bossbar'),
   supportedVersions,
   testedVersions,
-  supportFeature
+  supportFeature: (version, feature) => require('minecraft-data')(version).supportFeature(feature)
```
Comment (rom1504) [inline lib/loader.js:58]: can't do that here, version is not known at the top level

Comment (rom1504) [reply lib/loader.js:58]: ah you're taking it as arg ok

Comment (u9g) [reply lib/loader.js]: fixed

**Pull request merged**